.github/workflows/github-actions.yaml

```diff
@@ -0,0 +1,49 @@
+#checkov:skip:CKV2_GHA_1:Comment
+name: DevOps-Together CI
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+    
+permissions: read-all
+
+jobs:
+  linter:
+    runs-on: [ubuntu-22.04]
+    continue-on-error: true
```

Comment:
do we want to continue on errors?

Reply:
I added this because the pipeline would break at this stage.